### PR TITLE
[WIP] Adding remixd git plugin

### PR DESCRIPTION
**Pull request by edisinovcic**

### CODE REVIEW

File: src/remixAppManager.js [hunk 1]
```diff
@@ -214,6 +214,18 @@ export class RemixAppManager extends PluginEngine {
       version: '0.1.0-beta',
       location: 'sidePanel'
     }
+    const remixdgit = {
```
Comment (LianaHus) [inline src/remixAppManager.js]: Could you please use camel case here?

File: src/remixAppManager.js [hunk 2]
```diff
@@ -214,6 +214,18 @@ export class RemixAppManager extends PluginEngine {
...
+      name: 'Remixd.git',
+      displayName: 'Remixd git interface',
+      description: 'All git related stuff',
+      events: [],
+      methods: [],
+      url: 'remixd-git.surge.sh',
+      icon: 'https://git-scm.com/images/logos/1color-lightbg@2x.png',
```
Comment (LianaHus) [inline src/remixAppManager.js]: We need a quadratic icon. This is to show it on vertical icons panel
Something like https://images.app.goo.gl/gTQyYP7jhCVBSuL49

File: src/remixAppManager.js [hunk 3]
```diff
@@ -214,6 +214,18 @@ export class RemixAppManager extends PluginEngine {
       version: '0.1.0-beta',
       location: 'sidePanel'
     }
+    const remixdgit = {
+      name: 'Remixd.git',
```
Comment (LianaHus) [inline src/remixAppManager.js]: I'm not sure the bame can contain '.'
@GrandSchtroumpf ?

Comment (edisinovcic) [reply src/remixAppManager.js]: Will remove it completely :)

File: src/remixAppManager.js (hunk 1 above)
Comment (edisinovcic) [reply src/remixAppManager.js]: Sure

**Pull request closed without merging**